modules/runtime/nuxeo-runtime/src/main/java/org/nuxeo/runtime/model/impl/ComponentManagerImpl.java

```diff
@@ -501,7 +496,9 @@ public synchronized void unregisterServices(RegistrationInfo ri) {
 
     protected static void handleError(String message, Exception e) {
         log.error(message, e);
-        Framework.getRuntime().getMessageHandler().addWarning(message);
+        Framework.getRuntime()
+                 .getMessageHandler()
+                 .addMessage(Level.SEVERE, message, ComponentManagerImpl.class);
```

Comment:
Why not using `#getClass()` like in other changes?

Reply:
because it's a static method